content/pop-keyboard-shortcuts.md

```diff
@@ -70,12 +70,13 @@ Directional actions can use either the standard arrow keys or their Vim equivale
 | ------------------------------------------------------------------------------------------ | ---------------------------------------------- |
 | <kbd>SUPER</kbd> + <kbd>Ctrl</kbd> + <kbd>↑</kbd>/<kbd>↓</kbd>                               | Navigate between workspaces                    |
 | <kbd>SUPER</kbd> + <kbd>Home</kbd>/<kbd>End</kbd>                                            | Navigate to first/last workspace               |
-| <kbd>SUPER</kbd> + <kbd>Shift</kbd> + <kbd>Direction keys</kbd>                              | Move active window between workspaces/displays |
+| <kbd>SUPER</kbd> + <kbd>Shift</kbd> + <kbd>Direction keys</kbd>                              | Move active window between  upper and lower workspaces as well as right and left monitors respectively          |
+| <kbd>SUPER</kbd> + <kbd>Shift</kbd> + <kbd>Ctrl</kbd> + <kbd>↑</kbd>/<kbd>↓</kbd>            | Move the active window between upper and lower displays
 | <kbd>SUPER</kbd> + <kbd>ESC</kbd>                                                            | Lock the screen                                |
 
 ### Use the launcher
 
-The launcher allows searching through open windows and installed applications, and also has the additional functions listed below.
+The launcher allows searching through open windows and installed applications and has the additional functions listed below.
```

Comment:
While I appreciate the drive for conciseness, this change makes the "and"s run together and makes the sentence less clear. I think the old wording for this line was better.

If you want to make it more grammatically correct than it was originally, just add "it" after the "and" (following the comma) since this is an independent clause and should have a subject. (So it should read "and it also has..." after the comma.) Or simply separate it out into two sentences to get rid of the second "and" here.

Reply:
I'll fix it myself since it's been a while and the PR author doesn't seem to be around at the moment.